src/Core/Framework/MessageQueue/Middleware/RoutingOverwriteMiddleware.php

```diff
@@ -18,15 +17,9 @@ class RoutingOverwriteMiddleware implements MiddlewareInterface
 {
     /**
      * @param array<string, string|list<string>> $routing
-     * @param array<string, string|list<string>> $overwrite
      */
-    public function __construct(
-        /**
-         * @deprecated tag:v6.7.0 - Will be removed in v6.7.0.0. Use $overwrite instead
-         */
-        private readonly array $routing,
-        private readonly array $overwrite
-    ) {
+    public function __construct(private readonly array $routing)
```

Comment:
if named parameters are used, this change would be a break :see_no_evil: I am not sure, why we had both :thinking: I think "overwriting" was the better naming?

Reply:
This class is internal (it was only created a few months ago), the deprecated prop was taken over from the old messenger decorator to remind about removing, I think this is fine, wdyt ?

Reply:
yes :+1: